### PR TITLE
FILMO-0010 Update README.MD File

### PR DESCRIPTION
**Pull request by victorakd**
## Description
- Updated the README.MD file with FILMO project related information. 
- Added circleci budge.

## Type of change
Please select the relevant option

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?
Please describe the tests that you ran to verify your change
- [ ] End to end
- [ ] Integration
- [ ] Unit testing
- [x] N/A

## How to Manually Test
- Checkout to this branch
- Open README.MD file
- Hit (ctrl+shift+v) to README.MD file to preview badge

## Checklist:
- [x] My code follows the style guidelines of this project
- [ ] I have linked my code prior to submission
- [x] I have made corresponding changes to the documentation
- [ ] My changes generates no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# FILMO
+[![CircleCI](https://circleci.com/gh/Snow-Media-Technologies/filmo-backend/tree/develop.svg?style=svg)](https://circleci.com/gh/Snow-Media-Technologies/filmo-backend/tree/develop)
+
+FILMO description
+## External Dependencies
+This web application is written with the React framework using the Next.js library and a firebase database. You need React version 16.8 and above for the application to work
```
Comment (BenjaminKyamanywa) [inline README.MD]: @victorakd, I think we are using Postgres Database.

**Comment by BenjaminKyamanywa** [comment]
@victorakd there is a typo in your description for the word "budge"

File: README.MD [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+* Clone it.
+* Create your feature branch on your local machine with `git checkout -b your-feature-branch`
+* Push your changes to your remote branch with `git push origin your-feature-branch` ensure you avoid redundancy in your commit message.
+* Open a pull request to the `develop` branch and describe how your feature works
+* Refer to this wiki for proper [xxxxxx/xxxxxxxx/xx/xxx/x/x/x/x)
+## Contributors
+●  [Daniel Nuwagaba](xx/xxxx/xxx/xx/x/x)
+●  [Shema Eric](xx/xxxx/xxx/xx/x/x)
+●  [Aaron Sekisambu](xx/xxxx/xxx/xx/x/x)
+●  [Victor Asiimwe](xx/xxxx/xxx/xx/x/x)
```
Comment (BenjaminKyamanywa) [inline README.MD]: link to my github (https://github.com/BenjaminKyamanywa)

File: README.MD [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# FILMO
+[![CircleCI](https://circleci.com/gh/Snow-Media-Technologies/filmo-backend/tree/develop.svg?style=svg)](https://circleci.com/gh/Snow-Media-Technologies/filmo-backend/tree/develop)
+
+FILMO description
```
Comment (BenjaminKyamanywa) [inline README.MD]: the description seems to be missing.

File: README.MD [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# FILMO
+[![CircleCI](https://circleci.com/gh/Snow-Media-Technologies/filmo-backend/tree/develop.svg?style=svg)](https://circleci.com/gh/Snow-Media-Technologies/filmo-backend/tree/develop)
```
Comment (BenjaminKyamanywa) [inline README.MD]: coveralls badge also still pending as well.

**Comment by dnuwa** [comment]
@victorakd good work though you need to try and respond to feedback left on your PR as prompt as you can.

File: README.MD (hunk 2 above)
Comment (victorakd) [reply README.MD]: Thank you, your git has been added

**Pull request merged**